src/components/custom/_signup-public.js

```diff
@@ -132,12 +134,16 @@ const SocialButton = styled(Button)`
     height: 4rem;
     margin: 0 0.8rem;
     @media ${device.tabletL} {
-        width: 114px;
+        width: 14.25rem;
```

Comment:
I think you should express it in `px` here so that we know what the exact computed value is 😬